apps/avatax/integration/tests/product_without_taxes.spec.ts

```diff
@@ -0,0 +1,176 @@
+import { it } from "vitest";
+import { handler, spec } from "pactum";
+import { string } from "pactum-matchers";
+
+// Trick editor into highlighting gql, wihtout parsing
```

Comment:
Not sure if I understand, can you elaborate ?

Reply:
In order to make the string inside template literal be displayed in the editor as GraphQL syntax you need to use `gql` before the template literal (I suppose this is a requirement from Treesitter). I don't want to parse this string in any way (if I had imported the `gql` from `graphql-tag` it would parse the query into JSON of GraphQL AST), just get the syntax highlighting, thus this little hack to trick the editor

Reply:
So please add this comment to the code, if we are introducing hacks, lets deeply explain them.

Also by "editor" you probably mean *your* editor